src/helpers.ts

```diff
@@ -4,41 +4,39 @@
  *--------------------------------------------------------------------------------------------*/
 import * as fs from "fs";
 import * as JSONC from "jsonc-parser";
+import Mustache from "mustache";
 import * as log from "./Log";
+import ISettingsConfigJson from "./types/ISettingsConfigJson";
 
-/**
- * Loads a settings file and validates it with JSON schema, then returns it as a typed object.
- * @param serviceName The name of the service loading the settings. Used in log messages.
- * @param settingsFileName The path to the file to load.
- * @type T The type the settings should return as.
- */
-export function readSettings<T>(serviceName: string, settingsFileName: string): T {
-  let rawConfig: string;
+function readFile(filePath: string, fileType: string, serviceName: string): string {
```

Comment:
`serviceName` is a parameter `readFile`, `parseFile`, and `readSettings` but is in a different position in the parameter list across the three. Let's keep things tidy and consistent by ensuring the same parameter doesn't jump around:

- First parameter: `serviceName`
- Second parameter: `fileType`
- Third parameter: `filePath`

(Side note: I'm not wild about passing strings around to put in the log messages but I never came up with a better way to do it either.)

Reply:
Updated.